ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/PackageConfigurationFactory.kt

```diff
@@ -32,10 +34,22 @@ internal object PackageConfigurationFactory {
             return Result.failure(PackageConfigurationError("No packages found for ids $packageIdsInConfig"))
         }
         val packageInfos = filteredRCPackages.map {
+            val currentlySubscribed = when (it.packageType) {
+                PackageType.ANNUAL,
+                PackageType.SIX_MONTH,
+                PackageType.THREE_MONTH,
+                PackageType.TWO_MONTH,
+                PackageType.MONTHLY,
+                PackageType.WEEKLY,
+                -> activelySubscribedProductIdentifiers.contains(it.product.id)
+                PackageType.LIFETIME -> nonSubscriptionProductIdentifiers.contains(it.product.id)
+                PackageType.UNKNOWN, PackageType.CUSTOM -> false
```

Comment:
Should we move `PackageType.CUSTOM` above, so we also disallow purchasing for existing subscriptions in custom packages?